functions/task2.js

```diff
@@ -0,0 +1,16 @@
+//Засоби валідації
+function AgeVerification (age){
```

Comment:
names should written with camelCase, first letter should be in low register

Reply:
function name should answer the question - what to do. For functions which returns boolean value you an use simpler option.
So you can name it `getIsAdult` or `isAdult`

Reply:
your function does not return any value although it should